server/helpers/verifyEmailMessage.js

```diff
@@ -0,0 +1,13 @@
+const verifyEmailMessage = {
+  subject: 'Welcome to Author\'s Haven! Please Confirm your email',
+  body:
+    `<strong>
+      Click 
+        <a href="http://localhost:3001/api/test/verify">
```

Comment:
I like how you've abstracted the message out here, good job. However, shouldn't the link in the email contain some token as a query param perhaps? or how do we intend to verify the link was clicked by a particular user?

Reply:
I agree with @Darthrighteous . There should be a token in the link in order to know which account to verify.